gpiozero/output_devices.py

```diff
@@ -654,7 +642,56 @@ def is_active(self):
         return self.value != (0, 0, 0)
 
     is_lit = is_active
-    color = value
+
+    @property
+    def color(self):
+        """
+        Represents the color of the LED as a :class:`~colorzero.`Color` object.
+        """
+        return Color(*self.value)
+
+    @color.setter
+    def color(self, value):
+        if isinstance(value, Color):
+            self.value = value.rgb
+        else:
+            self.value = value
+
+    @property
+    def red(self):
+        return Red(self._leds[0].value)
+
+    @red.setter
+    def red(self, value):
+        self._stop_blink()
+        if isinstance(value, Red):
+            self.value = float(value)
```

Comment:
Surely this will blow up? In the value setter above it starts with `for component in value` ... but value is a float which isn't iterable. I don't think you need to test whether the value descends from the `Red` class at all (given `Red` is actually a sub-class of `float` anyway), unless you're intending to block assignment of `Green` and `Blue` values to the property, but this test won't actually do that anyway as they'll fall through to the `else` clause successfully?

Reply:
Yup, tests have shown that. I'll get them passing and give you another look.